src/go/rpk/pkg/cli/container/common/common.go

```diff
@@ -219,7 +215,7 @@ func CreateNetwork(c Client) (string, error) {
 		return "", err
 	}
 	if resp.Warning != "" {
-		log.Debug(resp.Warning)
+		fmt.Printf("Network creation warning: %v\n", resp.Warning)
```

Comment:
I figured this could be problematic enough to always print, since it affects container starting in general. I'm not sure why it was hidden behind debug logs previously.